zulipterminal/ui_tools/views.py

```diff
@@ -1939,14 +1939,21 @@ def __init__(
 
         # Get rendered message header and footer
         msg_box = MessageBox(message, controller.model, None)
+        msg_box.footer = [
```

Comment:
This changes the existing behavior, since the footer stores footlinks and reactions in message boxes.

Reply:
Yes I noticed that but it's still a widget_list that is being returned and since originally footer was an empty list in the `FullRawMsgView` so the earlier assertion passes so didn't need to change anything in `test_init`.

Adding two new assertions now to check if the widget_list element is urwid.Text and to check if the text is what it's supposed to be.